templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,14 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>
+      Manufacturer List
+      <a
+        style="float: center"
+        href="{% url 'taxi:manufacturer-create' %}"
+      >
```

Comment:
Do not break the tag in this way, place arguments on the same line with that tag.